ansible/roles/frontend/templates/apache2.conf

```diff
@@ -11,6 +11,12 @@
   RewriteCond %{HTTP:X-Forwarded-Proto} =http
   RewriteRule .* https://%{HTTP:Host}%{REQUEST_URI} [L,R=permanent]
 
+  # Security headers
+  Header always set Strict-Transport-Security "max-age=31536000; includeSubdomains;"
+  Header always set Cache-Control "max-age=31536000;"
```

Comment:
this will cause browsers to cache pages for a long time, and will cause confusion. setting such large cache times is useful for static content with URLs the change only based on their content (such as the our static assets), but for dynamic content this can make the system look broken as new data doesn't appear.

and Cache-Control is not really "security header" ...

Reply:
Good catch, also I didn't realize that all the other resources already had their cache-control set (by django maybe?), and I overlooked the global cache-control. I just pushed an update setting cache-control to `no-cache`, thus forcing the browser not to cache dynamic-generated content. This leaves all other `Cache-control` from assets unchanged, meaning each js/css/img has their own cache-control setting.